### PR TITLE
httprequest signals boostification

### PR DESCRIPTION
**Pull request by sima-fastly**
httprequest's signals updated from qt signals to boost. all the corresponding connections were also updated.

Tested locally

### CODE REVIEW

File: src/cpp/proxy/requestsession.cpp [hunk 1]
```diff
@@ -186,7 +186,11 @@ class RequestSession::Private : public QObject
 	bool autoShare;
 	XffRule xffRule;
 	XffRule xffTrustedRule;
-	bool isSockJs;
+	bool isSockJs;
```
Comment (jkarneges) [inline src/cpp/proxy/requestsession.cpp:189]: whitespace change?

File: src/cpp/proxy/requestsession.cpp [hunk 2]
```diff
@@ -1176,7 +1180,7 @@ RequestSession::RequestSession(DomainMap *domainMap, SockJsManager *sockJsManage
 }
 
 RequestSession::~RequestSession()
-{
+{
```
Comment (jkarneges) [inline src/cpp/proxy/requestsession.cpp:1183]: whitespace

File: src/cpp/proxy/sockjssession.cpp [hunk 3]
```diff
@@ -173,10 +175,6 @@ class SockJsSession::Private : public QObject
 
 	~Private()
 	{
-		keepAliveTimer->disconnect(this);
-		keepAliveTimer->setParent(0);
-		keepAliveTimer->deleteLater();
-
```
Comment (jkarneges) [inline src/cpp/proxy/sockjssession.cpp:179]: intentional?

File: src/cpp/proxy/updater.cpp [hunk 4]
```diff
@@ -79,7 +79,9 @@ class Updater::Private : public QObject
 	QTimer *timer;
 	ZhttpRequest *req;
 	Report report;
-	QDateTime lastLogTime;
+	QDateTime lastLogTime;
```
Comment (jkarneges) [inline src/cpp/proxy/updater.cpp:82]: whitespace?

File: src/cpp/proxy/sockjssession.cpp (hunk 3 above)
Comment (sima-fastly) [reply src/cpp/proxy/sockjssession.cpp]: thanks for catching this one!

**Review by jkarneges — APPROVED** [review]

**Pull request merged**